src/sentry/static/sentry/less/type.less

```diff
@@ -178,7 +178,7 @@ code {
 }
 
 pre {
-  word-break: keep-all;
+  word-break: break-all;
```

Comment:
Instead of changing this for all `pre` tags, can we scope it to `.event-details-container` in (`src/sentry/static/sentry/less/group-detail.less`)?